### PR TITLE
[CHANGE] Add cheats for DD and SSH

### PR DESCRIPTION
**Pull request by ImmortalPC**
I added some tips for DD and SSH

### CODE REVIEW

File: cheat/cheatsheets/ssh [hunk 1]
```diff
@@ -32,3 +32,8 @@ ssh user@example.com -C -c blowfish -X
 # Copy files and folders through ssh from remote host to pwd with tar.gz compression
 # when there is no rsync command available
 ssh user@example.com "cd /var/www/Shared/; tar zcf - asset1 asset2" | tar zxf -
+
+# Mount folder/filesystem through SSH
+# Install SSHFS from http://fuse.sourceforge.net/sshfs.html
+# Will allow you to mount a folder security over a network.
```
Comment (lucaswerkmeister) [inline cheat/cheatsheets/ssh]: s/security/securely/ ?

Also: The SourceForge link just redirects me to https://github.com/libfuse/sshfs…

Also might be worth mentioning that Emacs supports this directly:

    emacs /ssh:name@server/path/to/file

(Also works with `Host`s in SSH config, and `ssh:` can often be inferred)

Comment (ImmortalPC) [inline cheat/cheatsheets/ssh]: Oops. Typo and link have been corrected :)
Nice trick for emacs :)

**Comment by oliworx** [comment]
_output the sound from your microphone port to the ssh target computer's speaker port_ 
Really? A rather esoteric application of ssh!
We all know, we can combine unix commands to make cool things.
But on a cheat sheet I expect to see some every-day-examples.

**Comment by chrisallenlane** [comment]
@ImmortalPC, I agree with @oliworx in that the "sound over SSH" trick is probably a bit too esoteric for a day-to-day reference. Would you mind backing out that addition before I merge this in?

(FWIW, though, that is a really cool trick! :smile:)

**Comment by chrisallenlane** [comment]
Looks great. Thanks!

**Pull request merged**